README.md

```diff
@@ -13,6 +13,7 @@ Code and services for sending, receiving, and using astronomical alerts at OVRO.
 - slack_sdk
 - astropy
 - pygcn
+- gcn-kafka
 
 ## Design and Assumptions
 
```

Comment:
Could you also update the README to note that this version would use gcn-kafka instead of pygcn. 
Is there any pygcn in use? If not, we could remove that with this PR.

Reply:
We still use pygcn for LIGO alerts, so that's why it remains listed.